### ISSUE TITLE
Unable to specify utterances without multiples

### ISSUE THREAD

**Issue by stjohnjohnson** [matt-kruse/alexa-app#16]
I'm trying to just start a basic application that has a simple input.  If I don't provide multiple options per utterance, the `utterances()` function fails.
```
$ cat foo.js 
var alexa = require('alexa-app'),
    app = new alexa.app('foobar');

app.intent('HelloWorld',
    {
        "utterances": [
            "hello world"
        ]
    },
    function(request, response) {
        response.say("hello world");
    }
);

console.log(app.utterances());
console.log(app.schema());

$ node foo.js 

/tmp/node_modules/alexa-app/node_modules/js-combinatorics/combinatorics.js:287
        if (!arguments.length) throw new RangeError;
                                     ^
RangeError
    at cartesianProduct (/tmp/node_modules/alexa-app/node_modules/js-combinatorics/combinatorics.js:287:38)
    at generateUtterances (/tmp/node_modules/alexa-app/index.js:360:51)
    at /tmp/node_modules/alexa-app/index.js:265:17
    at Array.forEach (native)
    at utterances (/tmp/node_modules/alexa-app/index.js:264:30)
    at Object.<anonymous> (/tmp/foo.js:15:17)
    at Module._compile (module.js:456:26)
    at Object.Module._extensions..js (module.js:474:10)
    at Module.load (module.js:356:32)
    at Function.Module._load (module.js:312:12)
    at Function.Module.runMain (module.js:497:10)
    at startup (node.js:119:16)
    at node.js:935:3
```

**Comment by matt-kruse** [comment]
Just pushed a fix for this issue in version 2.1.2 
Thanks for the bug report. Embarassing. :)

**Comment by stjohnjohnson** [comment]
Thanks!!
